go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/golang/snappy v0.0.4
 	github.com/google/gopacket v1.1.19
 	github.com/gorilla/mux v1.8.1
-	github.com/grafana/dskit v0.0.0-20240528015923-27d7d41066d3
+	github.com/grafana/dskit v0.0.0-20240613153104-a26a26cbbb54
```

Comment:
this is same as used by https://github.com/grafana/mimir/pull/8365 for r293 backport